lisa/node.py

```diff
@@ -394,6 +395,9 @@ def find_partition_with_freespace(
                     else:
                         mountpoint = partition.mountpoint
 
+                    if df.get_filesystem_available_space(mountpoint, True) < size_in_gb:
```

Comment:
There is a check `if not partition.size_in_gb >= size_in_gb:` already, why need to check here again?

Reply:
this is to compare total size, not free size,  I will remove this, just check free size

Reply:
updated